tedana/workflows/tedana.py

```diff
@@ -177,6 +177,17 @@ def _get_parser():
                                 'use of IncrementalPCA. May increase workflow '
                                 'duration.'),
                           default=False)
+    optional.add_argument('--fittype',
+                          dest='fittype',
+                          action='store',
+                          choices=['loglin', 'curvefit'],
+                          help='Desired Fitting Method'
```

Comment:
The strings in this help message need trailing spaces in order to render properly.

Reply:
spaces added across the help text.